docs/Tarjan's Algorithm/Tarjan's.md

```diff
@@ -0,0 +1,159 @@
+---
+id: Tarjan's
+title: Introduction to Tarjan's Algorithm
```

Comment:
replace `title: Introduction to Tarjan's Algorithm` with `title: "Introduction to Tarjan's Algorithm"`